content/docs/ja/02-template-syntax.ja.md

```diff
@@ -245,7 +245,7 @@ each ブロックは `array.map(...)` のコールバックの第2数に相当
 
 ---
 
-*key* の式（各リストアイテムを一意に識別できる必要があります）が与えられた場合、データが変更されたときに Svelte は末尾にアイテムを追加したり削除するのではなくそれを使用して差分を取ります。key はどんなものでもよいですが、オブジェクト自体が変更されたときに同一性を維持できるため、文字列か数値をお勧めします。
+*key* の式（各リストアイテムを一意に識別できる必要があります）が与えられた場合、Svelte は、データが変化したときに（末尾にアイテムを追加したり削除するのではなく）キーを使用してリストの差分を取ります。key はどんなオブジェクトでもよいですが、そのオブジェクト自体が変更されたときに同一性を維持できるため、文字列か数値をお勧めします。
```

Comment:
- 和訳で「それ」が指すものが遠いので、指している「キー」にする
- listの訳し漏れを追加する
- 'when data changes'を原文のニュアンス通り、能動態として訳す
- 原文の'Svelte will use it to diff the list'というポイントが分かりやすくなるように、補足説明の', rather than...'を訳文では括弧にいれる
- このobjectは、一般的な「もの」ではなくて、JavaScriptの「オブジェクト」を意味していることを反映する